tools/releaseBuild/azureDevOps/WindowsBuild.yml

```diff
@@ -0,0 +1,513 @@
+jobs:
+
+- job: BuildJob
+  displayName: Build
+  condition: succeeded()
+  pool:
+    name: PowerShell
```

Comment:
@TravisEz13 do we still need to use our pool or we can use the `VS 2017` hosted pool now?

Reply:
still need to use our pool for windows builds